be/src/common/config.h

```diff
@@ -723,6 +723,11 @@ CONF_Int32(max_batch_publish_latency_ms, "100");
 // Config for opentelemetry tracing.
 CONF_String(jaeger_endpoint, "");
 
+#ifdef USE_STAROS
+CONF_String(starmgr_addr, "");
+CONF_Int32(starlet_port, "12345");
```

Comment:
Better to set a reasonable port

Reply:
The default port of starlet is assumed to be 8090. And can be modified in be.conf